share/eds/retriever_quadrant.go

```diff
@@ -45,7 +45,7 @@ type quadrant struct {
 	// source defines the axis for quadrant
 	// it can be either 1 or 0 similar to x and y
 	// where 0 is Row source and 1 is Col respectively
```

Comment:
[non-blocking] not sure this comment is still necessary, also maybe we could change the name of the field to axis as well?

Reply:
This comment is still correct if we look at the implementation detail of Axis, which is either 1 or 0.

To clarify, I could extend the opening comment. 
> source defines the axis to fetch the quadrant from

The reason it's called source is that all the shares are committed twice, in rows and cols(which you know), and we can get it from either source.

Reply:
its correct but the 1/0 is an impl detail that is not relevant to the understanding of its usage anymore
it can probably be simplified, but your choice

Reply:
Agree

Reply:
Updated